wce-ote/plugin/paratext.htm

```diff
@@ -55,21 +55,23 @@
 					<td><label id="fw_label" for="fw_type">{#paratext}</label>
 					</td>
 					<td colspan="3"><select id="fw_type" name="fw_type" onchange="checkstatus_typ()">
+							<option value="pageNum">{#fw_pagenumber}</option>
+							<option value="quireSig">{#fw_quiresignature}</option>
```

Comment:
Thanks for letting these be the default first few entries and for adding our Lect Bib Ref and Instruction